src/Common/src/Interop/Unix/System.Native/Interop.GetUid.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Also for this file, is this the right folder?

Reply:
Yes, this is fine.  GetEUid is in the Corelib version of this because it's also used there, but as this one isn't, here's good.